src/main/kotlin/no/nav/medlemskap/regler/v1/arbeidsforhold/JobberBrukerPaaNorskSkipRegel.kt

```diff
@@ -23,7 +24,9 @@ class JobberBrukerPaaNorskSkipRegel(
 
     override fun operasjon(): Resultat {
         return when {
-            arbeidsforhold sisteArbeidsforholdSkipsregister kontrollPeriodeForArbeidsforhold kunInneholder Skipsregister.NOR.name -> ja(regelId)
+            arbeidsforhold sisteArbeidsforholdSkipsregister kontrollPeriodeForArbeidsforhold kunInneholder Skipsregister.NOR.name ||
+                arbeidsforhold.skipsregisterErNISOgFartsområdeErInnenriks(kontrollPeriodeForArbeidsforhold)
+            -> ja(regelId)
```

Comment:
Tror regelen blir litt for "snill". Har sett litt på hvordan dataene ser ut for de med arbeidsforhold på skip, og de har gjerne flere arbeidsavtaler med forskjellige verdier på både skipsregister og fartsområde. Det betyr at et arbeidsforhold med to arbeidsavtaler hvor begge er NIS skip, men fartsområde er innenriks på den ene, og utenriks på den andre. I det tilfellet vil da regelen gi ja, men burde vært nei. Jeg kan forklare nærmere etter morgenmøte i morgen om ønskelig.

Det kan også være aktuelt å lage Cucumber test hvor det er forskjellige arbeidsavtaler med ulike verdier på skip under samme arbeidsforhold. Dersom man ser i securelogs på de som går ut på regel 7.1 kan man få inspirasjon til virkelighetsnære cucumbertester.